src/backend/prisma/schema.prisma

```diff
@@ -10,29 +7,122 @@ generator client {
-  published Boolean  @default(false)
-  author    User     @relation(fields: [authorId], references: [id])
-  authorId  Int
+enum CR_Type {
+  DESIGN_ISSUE
+  NEW_FUNCTION
+  OTHER
+  STAGE_GATING
+  WP_INITIATION
 }
 
-model Profile {
-  id     Int     @id @default(autoincrement())
-  bio    String?
-  user   User    @relation(fields: [userId], references: [id])
-  userId Int     @unique
+enum WBS_El_Status {
+  I
+  A
+  C
 }
 
 model User {
-  id      Int      @id @default(autoincrement())
-  email   String   @unique
-  name    String?
-  age     Int?
-  posts   Post[]
-  profile Profile?
+  user_id           Int                @id @default(autoincrement())
+  first_name        String
+  last_name         String
+  email_id          String             @unique
+  last_login        DateTime
+  role              Int
+  fab_weld_requests Fab_Weld_Request[]
+  change_requests   Change_Request[]
+  changes           Change[]
+  wbs_el_led        WBS_Element[]
+  projects_managed  Project[]
+}
+
+model Fab_Weld_Request {
+  fw_id            Int         @id @default(autoincrement())
+  date_submitted   DateTime
+  part_name        String
+  quantity         Int
+  resources_needed String
+  links            String
+  date_completed   DateTime
+  requestor        User        @relation(fields: [requestor_id], references: [user_id])
+  requestor_id     Int
+  wbs_el           WBS_Element @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id        Int
+}
+
+model Change_Request {
+  cr_id           Int         @id @default(autoincrement())
+  date_submitted  DateTime
+  type            CR_Type
+  what            String
+  why             String
+  timeline_impact DateTime
+  budget_impact   Int
+  scope_impact    String
+  docs            String
+  date_reviewed   DateTime
+  accepted        Boolean
+  changes         Change[]
+  requestor       User        @relation(fields: [requestor_id], references: [user_id])
+  requestor_id    Int
+  wbs_el          WBS_Element @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id       Int
+}
+
+model Change {
+  change_id        Int            @id @default(autoincrement())
+  attribute        String
+  original         String
+  new              Boolean
+  date_implemented DateTime
+  implementor      User           @relation(fields: [implementor_id], references: [user_id])
+  implementor_id   Int
+  change_request   Change_Request @relation(fields: [cr_id], references: [cr_id])
+  cr_id            Int
+}
+
+model WBS_Element {
+  wbs_el_id         Int                @id @default(autoincrement())
+  date_created      DateTime
+  wbs_num           String
+  wbs_el_name       String
+  wbs_el_status     WBS_El_Status
+  fab_weld_requests Fab_Weld_Request[]
+  change_requests   Change_Request[]
+  wbs_el_lead       User               @relation(fields: [wbs_el_lead_id], references: [user_id])
+  wbs_el_lead_id    Int
+}
+
+model Project {
+  project_id         Int            @id
```

Comment:
Primary key of supertype becomes primary key of subtype, is what I believe when remembering what Wolfgang said. So this key should be the same as its companion in the WBS_Element table. However, when looking at the spotify project, it seems like, we made this key something else and then made the supertype key a foreign key. I can't seem to know which one it is and am not sure when googling. This also applies to the Work Package table.